bve/src/parse/mesh/instructions/tests/mod.rs

```diff
@@ -62,6 +62,7 @@ fn generate_instructions_from_obj(input: &'static str) -> InstructionList {
 
     InstructionList {
         instructions: result,
+        warnings: Vec::default(),
```

Comment:
Why do you use `Vec::default()` here and `vec![]` in instruction_execution.rs?

Reply:
Because `Vec::new()`, `Vec::default()` and `vec![]` all do the same thing I have no idea what I should use.

Reply:
I literally use them all throughout the codebase. I should standardize them, but that's something for another PR.